ci/scripts/gitlab/artifactory_upload.sh

```
@@ -21,32 +21,13 @@ GITLAB_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd
 
 source ${GITLAB_SCRIPT_DIR}/common.sh
 
-GIT_TAG=$(get_git_tag)
-IS_TAGGED=$(is_current_commit_release_tagged)
+export GIT_TAG=$(get_git_tag)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)